lib/std/debug/SelfInfo.zig

```diff
@@ -792,10 +792,7 @@ pub const Module = switch (native_os) {
             _ = allocator;
             _ = address;
 
-            return switch (self.debug_data) {
-                .dwarf => |*dwarf| dwarf,
-                else => null,
-            };
+            return &(self.dwarf orelse return null);
```

Comment:
Personally I prefer `return if (self.dwarf) |*dwarf| dwarf else null;`, the double return is a bit harder to read